week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,8 @@
-var mentor1 = "Daniel";
-var mentor2 = "Irina";
-var mentor3 = "Mimi";
-var mentor4 = "Rob";
-var mentor5 = "Yohannes";
+function createPercentage(sts, mts) {
+var sumT = sts + mts
+var percSts = sts/sumT * 100;
+var percMts = mts/sumT * 100;
+    return percSts, percMts;
+ }
+ function greeting ()= createLongCreeting('Daniel', 30);
```

Comment:
this function is not related to the function you wrote at the top!
 
-You should have one function that calculates the percentage, and one function that creates a message
-the percentage should be rounded to the nearest whole number 
then console.log to see the result --> 
Percentage students: 65%
Percentage mentors: 35%